lib/viking/model/reflections/belongs_to_reflection.js

```diff
@@ -6,13 +6,11 @@ Viking.Model.BelongsToReflection = Viking.Model.Reflection.extend({
         this.options = _.extend({}, options);
     
         if (!this.options.polymorphic) {
-            this.modelName = this.options.model || {
-                className: name.camelize(),
-                singular:  name,
-                plural:    name.pluralize(),
-                routeKey:   name.split('/').pop().pluralize(),
-                paramKey:   name.split('/').pop()
-            };
+            if (typeof(this.options.modelName) === 'string') {
+                this.modelName = new Viking.Model.Name(this.options.modelName);
```

Comment:
Here did we want to pass the option as `objectName` and pass to the `Name` constructor with an underscore so `.Name(name)` always expects `name` to be underscored?